packages/gateway/src/services/l1proof.ts

```diff
@@ -0,0 +1,74 @@
+import {
```

Comment:
Rename file to l1Proof.ts or L1ProofService.ts

Reply:
it is within the services directory